docs/source/intro/run_docker.rst

```diff
@@ -49,56 +113,131 @@ This image contains a fully pre-configured AiiDA environment which makes it part
 
-         $ docker start -i aiida-container
+      .. tab-item:: Docker CLI
 
-      However, if you remove the container, **all data will be removed as well**.
+         Use the ``docker cp`` command if you need to copy files from your computer to the container or vice versa.
 
-      .. code-block:: console
+         For example, to copy a file named ``test.txt`` from your current working directory to the ``/home/aiida`` path in the container, run:
 
-         $ docker stop aiida-container
-         $ docker rm aiida-container
+         .. code-block:: console
 
-      The preferred way to persistently store data is to `create a volume <https://docs.docker.com/storage/volumes/>`__.
+            $ docker cp test.txt aiida-container:/home/aiida
 
-      To create a simple volume, run:
 
-      .. code-block:: console
+.. dropdown:: Persist data across different containers
+
+   The lifetime of the data stored in a container is limited to the lifetime of that particular container.
+
+   If you stop the container (`docker stop` or simply `Ctrl+D` from the container) and start it again, any data you created will persist.
+   However, if you remove the container, **all data will be removed as well**.
+
+   .. code-block:: console
+
+      $ docker rm aiida-container
+
+   The preferred way to persistently store data is to `create a volume <https://docs.docker.com/storage/volumes/>`__.
+
+   .. tab-set::
+
+      .. tab-item:: Docker Desktop
+
+         1. Open Docker Desktop
+         1. ???
+
+      .. tab-item:: Docker CLI
+
+         To create a simple volume, run:
+
+         .. code-block:: console
+
+            $ docker volume create container-home-data
+
+         Then make sure to mount that volume when the first time launching the aiida container:
+
+         .. parsed-literal::
+
+            $ docker run -it --name aiida-container -v container-home-data:/home/aiida aiidateam/aiida-core:latest bash
+
+         Starting the container with the above command, ensures that any data stored in the ``/home/aiida`` path within the container is stored in the ``conatiner-home-data`` volume and therefore persists even if the container is removed.
+
+         To persistently store the Python packages installed in the container, use `--user` flag when installing packages with pip, the packages will be installed in the ``/home/aiida/.local`` path which is mounted to the ``container-home-data`` volume.
+
+         You can also mount a local directory instead of a volume and to other container paths, please refer to the `Docker documentation <https://docs.docker.com/storage/bind-mounts/>`__ for more information.
+
+.. dropdown:: Backup the container
+
+   To backup the data of AiiDA, you can still follow the instructions in the `Backup and restore <backup_and_restore>`__ section.
+   However, the container provide a more convinient way if you just want to take a snapshot of the data by backup the whole container or the volume mounted to the container.
```

Comment:
However, Docker provides a convenient way to backup the container data by taking a snapshot of the entire container or the mounted volume(s).